mixer/template/sample/template.gen_test.go

```diff
@@ -969,12 +999,24 @@ func TestProcessReport(t *testing.T) {
 							Value:          "1",
 							Dimensions:     map[string]string{"s": "2"},
 							Int64Primitive: "54362",
+							DnsName:        `"myDNS"`,
+							Duration:       "request.duration",
+							EmailAddr:      `"myEMAIL"`,
```

Comment:
Should make this a valid address so validations will succeed here (if we use actual type mail.Address

Reply:
Will leave this as is since we are not using the mail.Address. And as soon as we use mail.Address, this test should fail.